model/Hardware/Properties/informationElementList.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# informationElementList
+
+## Summary
+
+Additional relevance information.
+
+## Description
+
+This elemtent provides relevant information to the product.
+
+## Metadata
+
+- name: informationElementList
+- Nature: ObjectProperty
+- Range::/Core/DictionaryEntry
```

Comment:
```suggestion
- Range: /Core/DictionaryEntry
```

Reply:
This is a minor typo. I will make change so the CI can further run and report other useful info for the development.

As agreed in previous Tech meeting (17 Dec 2024), we can just merge everything submitted, as it is the merge against the profile development branch (spdx:profile-hardware) and not to bottleneck the development process.